Analysis/output/country_data_report.Rmd

```diff
@@ -213,13 +213,13 @@ taxdat::plot_gam_fit_output_rates(config = params$config, cache = cache, cholera
 #### Ture underlying distribution of cases
 **True grid cases raster by time slices:**
 ```{r underlying distribution of cases,fig.cap=capFig("True underlying distribution of cases"), fig.height=5, fig.width=10}
-plot_true_grid_cases(config=params$config,cholera_directory=params$cholera_directory,cache=cache)
```

Comment:
I think you maybe didn't mean to commit the changes to this file?

Reply:
I think we should resolve the issue rather than commenting out the functions.

Reply:
These are the functions that should only be in the testing report. For some reasons that I do not know, they are in the country_data_report, and you suggested to remove them.

Reply:
I talked to Qulu and I think that the testing and non-testing reports got merged, and the testing plots should only happen if they are working. Can you try again (and let me and Qulu know if there are still problems with the reports)?